src/app/base/components/crypted-field/crypted-field.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="form-control">
+    <label class="text-gray-500 text-sm font-medium m-1">
+        Use crypted key for this project <em>- optional</em>
```

Comment:
frontend text can be password, just didn't want the field in the db to be called "password"

Reply:
also why is it "for this project"? I would expect Something like "Password:" as an optional input parameter for the component that can be overwrite it if necessary.

Reply:
Okay yes the default label will be Password, and we use that on the import and export of the records, but for project export it will be: "Encrypt zip file with password - optional" right?

Reply:
yes